### DISCUSSION TITLE
How to set the Display Template to display the image?

### DISCUSSION THREAD

**Discussion by cdwmhcc** [directus/directus#4861] [Archive]
![image](https://user-images.githubusercontent.com/8030134/113751493-96480180-973e-11eb-8e76-aef72714dd87.png)

This setting can not display the image, I have tried to select any lower-level fields of the file can not be displayed.

**Comment by cdwmhcc** [comment]
Did I miss something? Can I get help?

**Reply by cdanny358** [reply]
I have what I think is the same question. In Directus 8, images would show up as thumbnails in lists, like this: 
![Screen Shot 2021-04-14 at 11 14 21 AM](https://user-images.githubusercontent.com/390911/114759222-bfeec300-9d12-11eb-905f-2479f5dc768a.png)
In Directus 9 I can't seem to find any way to duplicate this functionality.

**Reply by benhaynes** [reply]
If you use the "image" display for the field then you will get a thumbnail.

**Reply by cdanny358** [reply]
Thanks. I think the issue is that there's no way I can see to change the display options after initial setup: 
![Screen Shot 2021-04-14 at 11 30 12 AM](https://user-images.githubusercontent.com/390911/114761085-ec0b4380-9d14-11eb-8f5b-7092e49e4812.png)
I had to go into phpMyAdmin and manually change the `display` type to `image`.

**Reply by benhaynes** [reply]
That's odd, any thoughts on this missing deselect @rijkvanzanten ?

**Reply by rijkvanzanten** [reply]
Must be a bug! Should be addressed in https://github.com/directus/directus/pull/3525 👍🏻

**Reply by abertay-university** [reply]
In the display template list, there is an option called **thumbnail**. That works for me
![image](https://user-images.githubusercontent.com/31690037/130651668-fdfb5c77-398f-48aa-871d-7ede6a93bafb.png)
